.github/workflows/build-test-package.yml

```diff
@@ -11,7 +11,6 @@ on:
       - main
 
 jobs:
-  timeout-minutes: 360
```

Comment:
Good catch, Matt! By the way, it looks like `timeout-minutes` is only supported for a specific job, not for all jobs together:

Looking at https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes

> **jobs.<job_id>.timeout-minutes**
>
> The maximum number of minutes to let a job run before GitHub automatically cancels it. Default: 360